src/css/tailwind.css

```diff
@@ -0,0 +1,5 @@
+.tailwind-layout {
```

Comment:
I would try adding `all: unset;` in here

btw I am having trouble running this repo. I do `yarn start` which works, but localhost:3000 hangs forever

Reply:
will try this! hmm. feels like a "did you power it on and off" question, but did you run `yarn` first?

Reply:
ya I did try that :/ its running smoothly, but not loading